pkg/plugin/aggregation/handler.go

```diff
@@ -97,6 +126,19 @@ func resultFromRequest(r *http.Request, muxVars map[string]string) *plugin.Resul
 	return result
 }
 
+func statusFromRequest(r *http.Request, muxVars map[string]string) (plugin.StatusUpdate, error) {
+	var update plugin.StatusUpdate
+	dec := json.NewDecoder(r.Body)
+	err := dec.Decode(&update)
+	update.Node = muxVars["node"]
```

Comment:
In the case where the decode fails and there is an error, is the resulting `update` object used?

Reply:
It shouldn't be (by convention if you get the err the value is not very meaningful), but the method is just a few lines long and avoids an unnecessary indentation by  just returning the object/wrapped error. We have to return an object anyways (even on the error case) so it doesn't matter if we do/dont set these values.